pep-0713.rst

```diff
@@ -0,0 +1,188 @@
+PEP: 713
+Title: Callable Modules
+Author: Amethyst Reese <amethyst at n7.gg>
+Sponsor: Łukasz Langa <lukasz at python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 20-Apr-2023
+Python-Version: 3.12
+
+
+Abstract
+========
+
+Modules are currently not directly callable. Classes can define a ``__call__``
+method that makes instance objects callable, but defining a similarly named
+function in the global module scope has no effect, and that function can
+only be called by importing or referencing it directly as ``module.__call__``.
+:pep:`562` added support for ``__getattr__`` and ``__dir__`` for modules, but
+defining ``__getattr__`` to return a value for ``__call__`` still does not
+make a module callable.
+
+This PEP proposes support for making modules directly callable by defining
+a ``__call__`` object in the module's global namespace, either as a standard
+function, or an arbitrary callable object.
+
+
+Motivation
+==========
+
+Many modules have only a single primary interface to their functionality.
+In many cases, that interface is a single callable object, where being able
+to use a simple import and using the module as a callable provides a more
+"pythonic" interface for users:
+
+.. code-block:: python
+
+    # user.py
```

Comment:
We could use the shorter version here and for the others, `python` is the default. (Plus capitalise Pythonic)

```suggestion
"Pythonic" interface for users::

    # user.py
```